python/tank/bootstrap/manager.py

```diff
@@ -516,7 +516,21 @@ def prepare_engine(self, engine_name, entity):
 
         return path
 
-    def get_pipeline_configurations(self, project):
+    def get_resolved_pipeline_configuration_descriptor(self, project_id=None):
```

Comment:
could this be symmetrical with `def get_pipeline_configurations(self, project):` and take a project dict (and no `None` default value)?